integration/app/lazy/list/list.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+import { Select } from '@ngxs/store';
+import { Observable } from 'rxjs';
+
+import { ListState } from '@integration/store/list/list.state';
```

Comment:
Do you think that we chould rather move the list state to be a lazy loaded state defined within this module?

Reply:
Yes